policy/release/git_branch/git_branch.rego

```diff
@@ -0,0 +1,27 @@
+package git_branch
+
+import rego.v1
+import data.lib
+
+# METADATA
+# title: Only allow builds from a trusted branch
+# description: Build must originate from a configured branch pattern (e.g., 'refs/heads/main')
+# custom:
+#   short_name: git_protected_branch_required
```

Comment:
A nitpick about the name here: There's nowhere in the check that we confirm that the branch is protected. So I'd rather not use "protected branch" in the name since that makes it seem like perhaps we are doing that.

How about calling it "expected_git_branch" or "trusted_git_branch".

Reply:
Renamed it to the simple git_branch